04.ls/ls.rb

```diff
@@ -1,15 +1,24 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
+require 'optparse'
+
 COLUMN_COUNT = 3
 
 def main
-  files = Dir.glob('*').sort_by(&:downcase)
+  params = ARGV.getopts('a')
```

Comment:
getopts = get optionsだと思うので、

```suggestion
  options = ARGV.getopts('a')
```

とした方が自然かも。

https://bootcamp.fjord.jp/pages/readable-variable-name

Reply:
ありがとうございます！
optionsを得るためだから`options`にするべきですね…。
修正いたしました！